### ISSUE TITLE
Cython not found

### ISSUE THREAD

**Issue by maxbachmann** [scikit-build/scikit-build#787]
I upgraded the `scikit-build` version in rapidfuzz for Linux from `0.14.1` to `0.16.0`. This leads to the following error:
```
  CMake Error at src/rapidfuzz/CMakeLists.txt:7 (find_package):
    By not providing "FindCython.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "Cython", but
    CMake did not find one.

    Could not find a package configuration file provided by "Cython" with any
    of the following names:

      CythonConfig.cmake
      cython-config.cmake

    Add the installation prefix of "Cython" to CMAKE_PREFIX_PATH or set
    "Cython_DIR" to a directory containing one of the above files.  If "Cython"
    provides a separate development package or SDK, be sure it has been
    installed.
  Call Stack (most recent call first):
    src/rapidfuzz/CMakeLists.txt:66 (create_cython_target)
```

**Comment by bnavigator** [comment]
Same as #786

**Comment by henryiii** [comment]
I think this is #786. I'm trying to get the release yanked (for another reason, no python_requires).

**Comment by henryiii** [comment]
I'll try to prepare a fix quickly and get a 0.16.1 out. Hopefully we'll be able to yank 0.16.0 as well.

**Comment by henryiii** [comment]
Fixed with 0.16.1.

**Comment by henryiii** [comment]
Is 0.16.1 working? You should be able to drop a few workarounds too.

**Comment by maxbachmann** [comment]
Yes it appears to work fine on my laptop. I still need to test on my desktop machine, which had some different problems with `0.15.0`.  Are you aware which workarounds can be dropped? I think I currently only have the following workarounds:
- the pypy workaround, which fixes the soabi
- setting `SKBUILD_LINK_LIBRARIES_KEYWORD`

**Comment by henryiii** [comment]
Those are the only two workarounds that can't be dropped. 😆 FindPython doesn't need special setting to find PyPy anymore. FindCython should work with FindPython now. And the discovery issue that broke some Conda builds is handled a different way.

**Comment by henryiii** [comment]
The SOABI fix is coming in PyPy 3.7.10; I'd rather like to see if we can also fix FindPython to handle the broken SOABI in PyPy < 3.7.10, but we'll see, and that takes longer anyway.
